<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtext.qrc">
    <normaloff>:/icons/16x16/appMono.png</normaloff>:/icons/16x16/appMono.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelSearch">
     <property name="text">
      <string>Search text:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboSearch"/>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkMatchCase">
         <property name="text">
          <string>&amp;Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkWholeWord">
         <property name="text">
          <string>&amp;Whole word</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkUseRegEx">
         <property name="text">
          <string>&amp;Regular expression</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioCurrentFile">
         <property name="text">
          <string>Current &amp;file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioCurrentFolder">
         <property name="text">
          <string>&amp;Current folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioAllFolders">
         <property name="text">
          <string>&amp;All folders</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboSearch</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../qtext.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
